lib/class/class_user_preferences.dart

```diff
@@ -328,6 +328,9 @@ class UserPreferences {
   }
 
   // unified Push settings
+  // the default value is to allow to reset the server configuration
+  final String fossPublicAlertServerUrlDefault = "https://alerts.kde.org";
```

Comment:
I don't fully understand what the difference is with `_fossPublicAlertServerUrl` 🤔 That's also marked as final and can thus not be edited and they have both the same values, so why use one over the other?